src/app/routes/api/v3/admin/forms/admin-forms.routes.ts

```diff
@@ -19,6 +20,9 @@ export const AdminFormsRouter = Router()
 AdminFormsRouter.use(withUserAuthentication)
 AdminFormsRouter.use(denyRpSpStudentEmails)
 
+// Log all non-get admin form actions
+AdminFormsRouter.use('/:formId([a-fA-F0-9]{24})/', logAdminAction)
```

Comment:
nit for style consistency
```suggestion
AdminFormsRouter.use('/:formId([a-fA-F0-9]{24})', logAdminAction)
```